spec/rails_helper.rb

```diff
@@ -30,6 +30,17 @@
   config.after type: :request do
     logout
   end
+
+  config.around(:each, testing_transactions: true) do |example|
+    DatabaseCleaner.allow_remote_database_url = true
+    self.use_transactional_tests = false
+    DatabaseCleaner.strategy = :truncation
+    DatabaseCleaner.clean
```

Comment:
This `DatabaseCleaner.clean` shouldn't be necessary, as the test should start with the database in a clean state. Is it necessary?